packages/scripts/eslint-preset.js

```diff
@@ -1,5 +1,8 @@
 module.exports = {
   extends: ['react-app'],
+  rules: {
+    'react/react-in-jsx-scope': 'off',
```

Comment:
seems like this rule isn't needed anymore so I turned it off because it's annoying